scripts/install_third_party_libs.py

```diff
@@ -89,6 +116,59 @@ def get_yarn_command():
     return 'yarn'
 
 
+def install_buf_and_protoc():
+    """Installs buf and protoc for Linux or Darwin, depending upon the
+    platform.
+    """
+    if os.path.exists(BUF_DIR):
+        return
```

Comment:
Is this check enough? What if some of the URL retrievals fails and we only install the stuff partially?

Reply:
Changed to files.